src/components/custom/_signup-new.tsx

```diff
@@ -110,88 +74,54 @@ const EmailButton = styled(Button)<{ isChecked?: boolean }>`
     @media ${device.tabletL} {
         margin-top: 24px;
     }
-
-    @media ${device.mobileL} {
-        font-size: 1.75rem;
-    }
-`
-const SignupWithContainer = styled.div`
-    display: flex;
-    justify-content: space-around;
-    flex-direction: row;
-    align-items: center;
-    margin-top: 2.4rem;
-
-    @media ${device.tabletL} {
-        margin-top: 4rem;
-    }
 `
-
 const SocialButton = styled(Button)<SocialButtonProps>`
     display: inline-flex;
     justify-content: center;
     align-items: center;
     box-shadow: none;
+    height: 4rem;
     background-color: ${(props) => props.bgColor || 'var(--color-white)'};
     border: solid 1px var(--color-grey-21);
-    width: 12.5rem;
-    height: 3.8rem;
     padding: 0.5rem 0;
 
-    &:hover {
-        background: ${(props) => {
-            if (props.provider === 'facebook') return 'var(--color-grey-4)'
-        }};
-    }
-
     @media ${device.tabletL} {
-        width: 100%;
-        height: 6rem;
-        margin-top: 1rem;
+        height: 4.8rem;
     }
-    @media ${device.mobileM} {
-        &:first-child {
-            margin-right: 1.2rem;
-        }
+`
+const SocialText = styled(Text)`
+    margin-right: 1.4rem;
+    margin-left: 0.7rem;
+    font-weight: 500;
+    font-size: 1.2rem;
+    color: ${(props) => props.color || 'var(--color-black-9)'};
+    @media ${device.tabletL} {
+        margin-left: 2.7rem;
+        font-size: 14px;
     }
 `
 const SocialWrapper = styled.div<CSSProperties>`
     width: 100%;
     margin-top: 2.4rem;
-    display: flex;
     justify-content: space-between;
-
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
     @media ${device.tabletL} {
-        flex-direction: column;
-        margin-top: 1rem;
-    }
-    @media ${device.mobileS} {
-        justify-content: space-between;
+        gap: 8px;
     }
 `
 const LoginText = styled(Text)`
```

Comment:
We can improve it a little bit
```suggestion
const LoginText = styled(Header)`
    text-align: center;
    font-weight: normal;
    align-self: center;
    margin-top: 2.6rem;
    
    @media ${device.tabletL} {
        font-size: 2rem;
    }
`
```
If you will refactor it, don't forget to add `as='p'` in `< LoginText />` 😄 
And we can remove mt and mb props because we didn't pass it anywhere after your changes